pkgs/development/python-modules/gower/default.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  fetchPypi,
+  scipy,
+  pandas,
+}:
+
+buildPythonPackage rec {
+
+  pname = "gower";
+  version = "0.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-NN21FY8Oi/ugk9yga5+Ie9okSZjRCvKjrYx0pu+htfY=";
+  };
+
+  patchPhase = ''
+    rm pyproject.toml
+  '';
+
+  pythonImportsCheck = [ "gower" ];
+
+  dependencies = [
+    setuptools
+    scipy
+    pandas
+  ];
+
+  meta = with lib; {
+    description = "Gower's distance calculation in Python.";
```

```suggestion
    description = "Gower's distance calculation in Python";
```
> Not end with a period (or any punctuation for that matter).

https://nixos.org/manual/nixpkgs/unstable/#var-meta-description